qemu/tests/balloon_check.py

```diff
@@ -355,6 +356,94 @@ def get_win_mon_free_mem(self, session):
         else:
             self.test.fail("Failed to get windows guest free memory")
 
+    def run_balloon_service(self, session, win_drive_letter):
```

Comment:
I think start_balloon_service() is better.

Reply:
This should be ok, I just want to keep consistent with the balloon service cmd: blnsrv.exe -r(means run)